tools/Xlsxtotabular/xls2tab.R

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env Rscript
-#Read xls/xlsx sheets - return tabular
-library(readxl)
 
-args = commandArgs(trailingOnly=TRUE)
+# Read xls/xlsx sheets - return tabular
+
+library(readxl)
 
-sheet<-read_excel(args[1],sheet=args[2])
-write.table(sheet,"out.tabular",sep="\t",row.names=FALSE)
\ No newline at end of file
+args = commandArgs(trailingOnly = TRUE)
+sheet <- read_excel(args[1],sheet = args[2])
+write.table(sheet,"out.tabular", sep = "\t", row.names = FALSE)

```

Comment:
can we also get rid of the quotes for every cell or are they needed?

Reply:
Yes... I had the same idea. I guess it is up to @yvanlebras. We can leave with it

Reply:
I just asked him, ok to remove the quotes, I'll do the commits :)